strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/domain/PypiMetadataDao.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.domain;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+public class PypiMetadataDao {
```

Comment:
Would you mind renaming this class to `PypiPackageInfo`?

Reply:
Changed.